feature-configs/hypershift-ci/fec/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../deploy/fec
```

Comment:
lets remove or put this folder in a comment that FEC was not tested for hypershift